F-Droid can't build: 'Minimum supported Gradle version is 5.1.1'

Ref:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/vagrant/build/de.csicar.mensaplan/app/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin [id 'com.android.application']
   > Minimum supported Gradle version is 5.1.1. Current version is 4.10.1. If using the gradle wrapper, try editing the distributionUrl in /home/vagrant/build/de.csicar.mensaplan/app/gradle/wrapper/gradle-wrapper.properties to gradle-5.1.1-all.zip
```

I've fixed it now in the [metadata](https://gitlab.com/fdroid/fdroiddata/commit/3ff88fabf2f621854c39ec874639742b8550b9ba) but maybe you want a more permanent fix.

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip

